deepmd/dpmodel/descriptor/se_e2_a.py

```diff
@@ -313,6 +327,17 @@ def call(
 
     def serialize(self) -> dict:
         """Serialize the descriptor to dict."""
+        if not self.type_one_side and self.exclude_types:
+            for embedding_idx in itertools.product(range(self.ntypes), repeat=2):
+                # not actually used; to match serilization data from TF to pass the test
+                if embedding_idx in self.emask:
+                    for ilayer in range(len(self.neuron)):
+                        layer = self.embeddings[embedding_idx][ilayer]
+                        layer.w.fill(0.0)
+                        layer.b.fill(0.0)
+                        if layer.idt is not None:
+                            layer.idt.fill(0.0)
+
```

Comment:
ugly code. can tf avoids such serialization?

Reply:
or the hacking code should be moved to network collection?

Reply:
why we did not see the hack in pt...

Reply:
PT doesn't compare with TF directly.

Reply:
DP and PT will store this network, although it is never used. TF will lose the information after deserializing + serializing.

Reply:
I move it to `Network.clear()`. See b3160c878aaed8187cc60589bdcd70f7be872dda.